### DISCUSSION TITLE
Open source tiny rant...

### DISCUSSION THREAD

**Discussion by petr24** [rgossiaux/svelte-headlessui#140] [General]
I've stumbled across this project a few weeks ago searching for a way to integrate tailwind UI with Svelte, as I'm sure others have. Comparing this to Headless UI which is made by a team or person (who gets paid), this open source version is damn good especially considering (free). 

However just browsing the issues page, and personally having committed zero effort to the project, it bothers my that I see nothing but a patten of questions that have a sense of entitlement. "When is X going to be ready", "Is this being updated...", etc. Open source projects are supposed to be community based, @rgossiaux is not the only person who is supposed code everything by himself for free, and the only thing he gets in return for doing this project is more tickets to code or fix for free! 

Open source is about finding a bug or a missing feature, and then posting a solution... when did open source become outsourced free labor?

We are all adults with real lives and responsibilities... let's contribute, not ask. 

For my part I don't have a bunch of time, but I can help with documentation examples. The documentation is great, but the example code is a non-styled example and does not match up with the styled component in the demo. It would be great to have the code of the styled component that is being displayed. This way you can copy and paste straight from demo and see the component in action as presented in the demo. @rgossiaux would this be valuable add?

**Comment by HemalR** [comment]
Echo the sentiments 👍

Just thinking out loud - this may be against TailwindUI's terms of service.  I.e. - Unless subscribed to them, it is not possible to copy/paste the code from them either and having their component library code available here in Svelte would not be fair on them.

I do think the docs are great with regards to explaining how to port over the code from e.g. React though. 👍

@rgossiaux Thanks for all the work you have put in here!

**Comment by rolanday** [comment]
+1 to ^^. Svelte would be a non-starter for me otherwise. I'm hooked on tailwinds (have the full license) and coming from react world. Thank you so much!! @rgossiaux rocks!
